### PR TITLE
Remove isRoomClean field from add command

### PR DESCRIPTION
**Pull request by bryanhce**

### CODE REVIEW

**Review by Lan-Jingbo — APPROVED** [review]
Very clear code/content editing
LVeryGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Format: `list`
 
 Edits an existing guest in the guest book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] [rc/IS_ROOM_CLEAN] ​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] ​`
```
Comment (JustinPeng13) [inline docs/UserGuide.md]: Could we remove the space at the end of the code block?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -221,14 +221,14 @@ _Details coming soon ..._
...
-| **List**             | `list`                                                                                                                                                                                                  |
-| **Help**             | `help`                                                                                                                                                                                                  |
+| Action               | Format, Examples                                                                                                                                                                   |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**              | `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS ​` <br> e.g., `add n/John Doe p/98765432 e/johnd@example.com rm/ 05-73 dr/19/05/20 - 24/05/22 ng/3`   |
```
Comment (JustinPeng13) [inline docs/UserGuide.md]: Could we remove the space at the end of the code block? (ng/NUMBER_OF_GUESTS `)

File: guestbook.json [hunk 3]
```diff
@@ -9,5 +9,15 @@
...
+  }, {
+    "name" : "Juuu",
+    "phone" : "98765432",
+    "email" : "johnd@example.com",
+    "room" : "05-73",
+    "dateRange" : "13/09/22 - 15/09/22",
+    "numberOfGuests" : "1",
+    "isRoomClean" : "yes",
+    "bill" : "0.00",
+    "request" : "-"
```
Comment (JustinPeng13) [inline guestbook.json]: What is this entry for?

File: src/main/java/seedu/guest/model/guest/IsRoomClean.java [hunk 4]
```diff
@@ -10,26 +10,51 @@
...
-     * @param isClean A valid is room clean.
+     * @param isRoomClean A valid isRoomClean.
      */
-    public IsRoomClean(String isClean) {
-        requireNonNull(isClean);
-        checkArgument(isValidIsRoomClean(isClean), MESSAGE_CONSTRAINTS);
-        value = isClean;
+    public IsRoomClean(String isRoomClean) {
+        requireNonNull(isRoomClean);
+        String isRoomCleanToLower = isRoomClean.toLowerCase();
```
Comment (JustinPeng13) [inline src/main/java/seedu/guest/model/guest/IsRoomClean.java]: Is this unnecessary as `isValidIsRoomClean` already does it?

File: src/main/java/seedu/guest/model/guest/IsRoomClean.java [hunk 5]
```diff
@@ -10,26 +10,51 @@
...
+        requireNonNull(status);
+        assert isValidIsRoomClean(status);
+
+        String standardisedRoomUnclean = "no";
+        String standardisedRoomClean = "yes";
+        String abbreviatedRoomUnclean = "n";
+        String unabbreviatedRoomUnclean = "no";
+
+        if (status.equals(abbreviatedRoomUnclean) || status.equals(unabbreviatedRoomUnclean)) {
+            return standardisedRoomUnclean;
```
Comment (JustinPeng13) [inline src/main/java/seedu/guest/model/guest/IsRoomClean.java:53]: Aren't `standardisedRoomUnclean` and `unabbreviatedRoomUnclean` the same? Can we eliminate one of these variables?

File: src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java [hunk 6]
```diff
@@ -57,45 +52,40 @@ public void parse_allFieldsPresent_success() {
...
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_AMY + EMAIL_DESC_BOB
-                + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB,
                 new AddCommand(expectedGuest));
 
         // multiple rooms - last room accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
                         + ROOM_DESC_AMY + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
-                        + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                        + NUMBER_OF_GUESTS_DESC_BOB ,
```
Comment (JustinPeng13) [inline src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java]: Can we remove the extra space here?

**Review by JustinPeng13 — CHANGES_REQUESTED** [review]
Just a few formatting issues and clarifying questions!

File: docs/UserGuide.md (hunk 1 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Fixed!

File: guestbook.json (hunk 3 above)
Comment (bryanhce) [reply guestbook.json]: This was a new entry I created when I was manually testing the edit command for isRoomClean.

File: src/main/java/seedu/guest/model/guest/IsRoomClean.java (hunk 4 above)
Comment (bryanhce) [reply src/main/java/seedu/guest/model/guest/IsRoomClean.java]: Yes, you are right. However, isValidRoomClean does not return the lowercased string. So I added the toLowerCase method to operate on the string in the standardiseIsRoomClean instead.

File: src/main/java/seedu/guest/model/guest/IsRoomClean.java (hunk 5 above)
Comment (bryanhce) [reply src/main/java/seedu/guest/model/guest/IsRoomClean.java:53]: Yes, you are right. Initially, I had both as they served different purposes, but I think reusing the variables in this instance works.

File: src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java (hunk 6 above)
Comment (bryanhce) [reply src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java]: Fixed!

File: docs/UserGuide.md (hunk 2 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Fixed!

**Review by JustinPeng13 — APPROVED** [review]
lgtm!

**Pull request merged**